ruff.toml

```diff
@@ -5,6 +5,10 @@ exclude = [
 
 line-length = 119
 
+[format]
```

Comment:
Why we need this line? `git config --global core.autocrlf true` would be better and superior solution.

Reply:
Reason: github CI check for CRLF
TODO: find CI check for all text files, not only python